payments-core/src/main/java/com/stripe/android/CardBrandFilter.kt

```diff
@@ -0,0 +1,16 @@
+package com.stripe.android
+
+import com.stripe.android.model.CardBrand
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+interface CardBrandFilter : Parcelable {
```

Comment:
I'm guessing this is purely internal stripe API, not intended to be public. In which case, you'll need to make this `@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)`

Reply:
Yeah definitely should not be public, thanks for the flag. Surprised that's needed tbh.